Sources/Swinject/Resolver.swift

```diff
@@ -12,7 +12,7 @@
 //
 
 
-public protocol Resolver {
+public protocol Resolver: AnyObject {
```

Comment:
Just to make it so we could have weak references to the Swinject Resolver in rare cases